test/controllers/sections/firstTransporter/FirstTransporterReviewControllerSpec.scala

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.sections.firstTransporter
+
+import base.SpecBase
+import controllers.actions.{FakeDataRetrievalAction, FakeMovementAction}
+import forms.sections.firstTransporter.FirstTransporterReviewFormProvider
+import mocks.services.MockUserAnswersService
+import models.UserAnswers
+import models.requests.DataRequest
+import models.sections.ReviewAnswer
+import models.sections.ReviewAnswer.KeepAnswers
+import navigation.FakeNavigators.FakeFirstTransporterNavigator
+import pages.sections.firstTransporter.FirstTransporterReviewPage
+import play.api.data.Form
+import play.api.i18n.Messages
+import play.api.mvc.{AnyContentAsEmpty, Call}
+import play.api.test.Helpers._
+import play.api.test.{FakeRequest, Helpers}
+import viewmodels.checkAnswers.sections.firstTransporter.FirstTransporterCheckAnswersHelper
+import views.html.sections.firstTransporter.FirstTransporterReviewView
+
+import scala.concurrent.Future
+
+class FirstTransporterReviewControllerSpec extends SpecBase with MockUserAnswersService {
+
+  lazy val formProvider = new FirstTransporterReviewFormProvider()
+  lazy val form: Form[ReviewAnswer] = formProvider()
+  lazy val view: FirstTransporterReviewView = app.injector.instanceOf[FirstTransporterReviewView]
+  lazy val helper: FirstTransporterCheckAnswersHelper = app.injector.instanceOf[FirstTransporterCheckAnswersHelper]
+  val submitCall: Call = controllers.sections.firstTransporter.routes.FirstTransporterReviewController.onSubmit(testErn, testArc)
+
+  class Test(val userAnswers: Option[UserAnswers] = Some(emptyUserAnswers)) {
```

Comment:
The default value `Some(emptyUserAnswers))` looks like it can be removed. That or the tests that use it don't have to pass `Some(emptyUserAnswers)` in

Reply:
Resolved in [7f6c69e](https://github.com/hmrc/emcs-tfe-change-destination-frontend/pull/19/commits/7f6c69ed0728afcb9027fdf576b63dec3c287e2f)